packages/react-dom/server.bun.js

```diff
@@ -11,8 +11,22 @@
 import ReactVersion from 'shared/ReactVersion';
 export {ReactVersion as version};
 
+export function renderToString() {
+  return require('./src/server/ReactDOMLegacyServerBrowser').renderToString.apply(
+    this,
+    arguments,
+  );
+}
+
+export function renderToStaticMarkup() {
+  return require('./src/server/ReactDOMLegacyServerBrowser').renderToStaticMarkup.apply(
+    this,
+    arguments,
+  );
+}
+
```

Comment:
We should add the other two. you can see how we handle this with .browser.js here

https://github.com/facebook/react/blob/main/packages/react-dom/src/server/ReactDOMLegacyServerBrowser.js#L42-L54

basically we want all exports to be exported on all variants so if you end up using the wrong API in the wrong environment you get a useful error rather than some invalid import error.